### PR TITLE
Add --version option to the main command

### PR DESCRIPTION
**Pull request by TomasHofman**

### CODE REVIEW

File: prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/MainCommand.java [hunk 1]
```diff
@@ -45,4 +51,11 @@ public Integer call() {
         return ReturnCodes.INVALID_ARGUMENTS;
     }
 
+    static class VersionProvider implements CommandLine.IVersionProvider {
+        @Override
+        public String[] getVersion() throws Exception {
+            Manifest manifest = new Manifest(getClass().getClassLoader().getResourceAsStream("META-INF/MANIFEST.MF"));
```
Comment (spyrkob) [inline prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/MainCommand.java:57]: I think that'll only work if Prospero is distributed as a single shaded jar, not eg. when using jboss-modules

**Review by spyrkob — CHANGES_REQUESTED** [review]

File: prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/MainCommand.java (hunk 1 above)
Comment (TomasHofman) [reply prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/MainCommand.java:57]: That's probably true, I didn't thought of the jboss-modules distribution. I will look into that.

Comment (TomasHofman) [reply prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/MainCommand.java]: I updated the PR, the VersionProvider now goes through all the MANIFESTs to locate the prospero-cli manifest and gets the version from there.

**Review by spyrkob — APPROVED** [review]
LGTM, thanks!

**Pull request merged**